pyoos/utils/__init__.py

```diff
@@ -0,0 +1,9 @@
+from __future__ import (absolute_import, division, print_function)
+
+from .etree import ElementType, ParseError, etree
+
+__all__ = [
+    'ElementType',
+    'ParseError',
+    'etree'
```

Comment:
This is the most important change in the latest commit. It introduces these three in `pyoos.utils` so we can use either `lxml` or `xml`.